apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClient.cs

```diff
@@ -76,6 +76,11 @@ public enum Reply
     /// </summary>
     public static TimeSpan DefaultAckDeadline { get; } = TimeSpan.FromSeconds(60);
 
+    /// <summary>
+    /// The default message Acknowledgement deadline of 60 seconds for exactly-once delivery subscriptions.
```

Comment:
Let's work out whether we want it to be "Acknowledgement", "ACKnolwedgement" or "acknowledgement" - in lines 80, 85 and 90 we've got all three...

Reply:
Made it consistent with "acknowledgement" across the public API docs.